test/UnitInsertDelete.cpp

```diff
@@ -52,7 +52,7 @@ void getPartHashCodes(const std::string& testname, const std::string& response,
     TST_LOG("Status reports " << totalParts << " parts.");
 
     Poco::RegularExpression endLine("[^\n\r]+");
-    Poco::RegularExpression number("^[0-9]+$");
+    Poco::RegularExpression number("^-?[0-9]+$");
```

Comment:
this is not needed, it will be fixed soon: https://gerrit.libreoffice.org/c/core/+/169691

Reply:
I don't see why. The change didn't make it unsigned, as far as I see.

Reply:
Ah, right. I looked at getId(), not getUniqueId() -> needs to be changed

Reply:
So please let's not block on this. If a future change makes them unsigned, this test may be reverted to the previous state, maybe with an explicit comment requiring them to be unsigned.